api_tests/caching/test_caching.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import unicode_literals
+import copy
+
+import requests
+from django.conf import settings
+from requests.auth import HTTPBasicAuth
+
+
+class TestVarnish(object):
+
+    local_varnish_base_url = 'http://localhost:8193/v2/'
+    local_python_base_url = 'http://localhost:8000/v2/'
+
+    def setUp(self):
+        if not settings.ENABLE_VARNISH:
+            return
+        self.authorization = HTTPBasicAuth('mocha@osf.io', 'password')
+
+    def test_compare_python_responses_to_varnish_responses(self):
+        if not settings.ENABLE_VARNISH:
```

Comment:
Minor: You can use `unittest.skipif(not settings.ENABLE_VARNISH, '...')` so that the test output will show that this test was explicitly skipped.